src/components/scenes/SendScene.js

```diff
@@ -304,7 +304,7 @@ class SendComponent extends React.PureComponent<Props, State> {
           <ButtonsModal
             bridge={bridge}
             title={s.strings.fio_no_bundled_err_msg}
-            message={s.strings.fio_no_bundled_non_fio_err_msg}
+            message={`${s.strings.fio_no_bundled_non_fio_err_msg} ${s.strings.fio_no_bundled_add_err_msg}`}
```

Comment:
Do not add a second string, just modify the first one.

Reply:
Please confirm this change. There are two strings here because second is used in other places.